### ISSUE TITLE
KBFS - Input/output error after changing accounts

### ISSUE THREAD

**Issue by jms1voalte** [keybase/client#16671]
Earlier today I signed out of my personal keybase account and created a new one, just for this laptop (i.e. just for work, so I can separate work and personal stuff). I had to fully quit and restart the app (macOS) in order for KBFS to start working, and things have been working as expected for the past few hours, but I just tried to save a file in one of the team folders and it came back with `Input/output error`.

On the work laptop (where this issue happened), I ran `du -sh /keybase/team/*` to make it scan everything. There are two team folders exhibiting this behaviour (you should see the names in the log comments.)

On my personal laptop (also macOS, signed into my personal Keybase account, which is also an owner/admin of the same set of teams) the same `du -sh /keybase/team/*` command worked as expected, and showed disk usage numbers in the ranges I expected.

my log id: 442894da323c144d5e92641c

**Comment by strib** [comment]
The service is returning this error to KBFS:

```
You don't have access to KBFS for this team libkb.KeyMaskNotFoundError
```

even though on our end, it looks like the user is an admin of the team (though the admin-ship was added just a few hours earlier, so maybe there's a caching issue somewhere?).

@mmaxim seems to have written this code, but is out this week.  @maxtaco who else can look at this?  In the log, see "FID=rHdxD62UnDv8z3hX2AsCrA" as an example.

**Comment by jms1voalte** [comment]
Yeah, the new "work" user has the same owner/admin permissions that my "personal" user does ... the idea is, once I've verified that the work user is able to do everything it needs to do, I'm going to remove my personal user from the work-related teams.

I'm about to leave anyway, I'll try this again tomorrow. Thanks for looking into it.

**Comment by strib** [comment]
@mlsteele can you please take an initial look?

**Comment by maxtaco** [comment]
@mlsteele we'll track with a JIRA ticket, no need.

**Comment by maxtaco** [comment]
@jms1voalte  can you try to quit the app, `keybase db nuke`, then start the app back up?

**Comment by jms1voalte** [comment]
Actually, "modalduality" caught me via chat and had me run `keybase team profile-load xxx` for each of the teams I was having trouble with, and that seems to have fixed the issue.

I can still do a nuke if you need me to, but at the moment it doesn't seem necessary.

**Comment by maxtaco** [comment]
@jms1voalte glad to hear it, no need for further futzing. We'll investigate on our end, thanks!
